.gitignore

```diff
@@ -1,3 +1,2 @@
 .idea/
 .vscode/
-vendor/
```

Comment:
Is there a reason to check in the vendor tree?

Reply:
The main reason is that it is not convenient for some developers to install the vendor due to some network reasons and in other operators they always has the vendor,  so I think we should add the vendor to the code tree.

Reply:
LGTM for vender tree.